### PR TITLE
[zh-cn]: update the translation of HTTP `Access-Control-Request-Headers` header

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/http/headers/access-control-request-headers/index.md [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
-| 报头类型                              | {{Glossary("Request header")}} |
-| ------------------------------------- | ------------------------------ |
-| {{Glossary("Forbidden header name")}} | yes                            |
+**`Access-Control-Request-Headers`** 请求标头用于浏览器在发起{{glossary("preflight request", "预检请求")}}时告知服务器客户端在实际请求发送时可能附带的 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)（例如使用 {{domxref("fetch()")}} 或 {{domxref("XMLHttpRequest.setRequestHeader()")}}）。服务器端对应的响应标头为 {{HTTPHeader("Access-Control-Allow-Headers")}}，表示回应浏览器的请求标头信息。
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">标头类型</th>
+      <td>{{Glossary("Request header")}}</td>
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/access-control-request-headers/index.md:16]:
```suggestion
      <td>{{Glossary("Request header", "请求标头"}}</td>
```

File: files/zh-cn/web/http/headers/access-control-request-headers/index.md [hunk 2]
```diff
@@ -1,31 +1,42 @@
...
-Access-Control-Request-Headers: <header-name>, <header-name>, ...
+```http
+Access-Control-Request-Headers: <header-name>,<header-name>,…
 ```
 
 ## 指令
 
 - \<header-name>
-  - : 在实际请求中将要包含的一系列 [HTTP](/zh-CN/docs/Web/HTTP/Headers) 头，以逗号分隔。
+  - : 以逗号分隔包含了实际请求中一系列的 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)。
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/access-control-request-headers/index.md:34]:
```suggestion
  - : 以逗号分隔的、小写、排序、独一无二 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)列表包含在请求中。
```

File: files/zh-cn/web/http/headers/access-control-request-headers/index.md [hunk 3]
```diff
@@ -1,31 +1,42 @@
...
 ---
 
 {{HTTPSidebar}}
 
-请求头 **`Access-Control-Request-Headers`** 出现于 {{glossary("preflight request")}}（预检请求）中，用于通知服务器在真正的请求中会采用哪些请求头。
-
-| 报头类型                              | {{Glossary("Request header")}} |
-| ------------------------------------- | ------------------------------ |
-| {{Glossary("Forbidden header name")}} | yes                            |
+**`Access-Control-Request-Headers`** 请求标头用于浏览器在发起{{glossary("preflight request", "预检请求")}}时告知服务器客户端在实际请求发送时可能附带的 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)（例如使用 {{domxref("fetch()")}} 或 {{domxref("XMLHttpRequest.setRequestHeader()")}}）。服务器端对应的响应标头为 {{HTTPHeader("Access-Control-Allow-Headers")}}，表示回应浏览器的请求标头信息。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-request-headers/index.md:10]:
```suggestion
**`Access-Control-Request-Headers`** 请求标头用于浏览器在发起{{glossary("preflight request", "预检请求")}}时告知服务器客户端在实际请求发送时（例如使用 {{domxref("fetch()")}} 或 {{domxref("XMLHttpRequest.setRequestHeader()")}}）可能附带的 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)。服务器端对应的响应标头为 {{HTTPHeader("Access-Control-Allow-Headers")}}，表示对浏览器的请求标头的回应。
```

File: files/zh-cn/web/http/headers/access-control-request-headers/index.md [hunk 4]
```diff
@@ -1,31 +1,42 @@
...
-Access-Control-Request-Headers: <header-name>, <header-name>, ...
+```http
+Access-Control-Request-Headers: <header-name>,<header-name>,…
 ```
 
 ## 指令
 
 - \<header-name>
-  - : 在实际请求中将要包含的一系列 [HTTP](/zh-CN/docs/Web/HTTP/Headers) 头，以逗号分隔。
+  - : 以逗号分隔、小写、排序、唯一 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)列表包含在请求中。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-request-headers/index.md:34]: 这一段的翻译请好好改改

Comment (T34-active) [inline files/zh-cn/web/http/headers/access-control-request-headers/index.md:34]:
```suggestion
  - : 在请求中包含的唯一的、以逗号分隔的小写 [HTTP 标头](/zh-CN/docs/Web/HTTP/Headers)的排序列表。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**